Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -136,7 +136,8 @@ class AppSettingsTableViewController: SettingsTableViewController {
                 SlowTheDatabase(settings: self),
                 ForgetSyncAuthStateDebugSetting(settings: self),
                 SentryIDSetting(settings: self),
-                ChangeToChinaSetting(settings: self)
+                ChangeToChinaSetting(settings: self),
+                ToggleOnboarding(settings: self)
```

Comment:
**Debub Setting:** Allows user to toggle between default and version2 of the intro screen. Toggle them and then show the tour